services/bonita-time-tracker/src/test/java/org/bonitasoft/engine/tracking/RecordAssert.java

```diff
@@ -0,0 +1,121 @@
+package org.bonitasoft.engine.tracking;
```

Comment:
this utility test file is generated by maven assertJ assertion plugin. Is header mandatory for this kind of file ?